### ISSUE TITLE
Cropping without showing the UI?

### ISSUE THREAD

**Issue by Mehdi-Sohrabi0** [guoyingtao/Mantis#155]
Hi,

Is it possible to crop an image without showing the UI? By preserved `Transformation` information.

**Comment by guoyingtao** [comment]
Currently, it is not supported by Mantis.
It might be possible, but I need to take a look at it later.

**Comment by Mehdi-Sohrabi0** [comment]
Hi @guoyingtao 
Thank you for your answer.

Currently I'm opening an transparent `CropController` and get the result back from it, but it doesn't feel good to do that.
So I would appreciate if you can spend some time on this.

````swift
cropViewController.modalPresentationStyle = .overCurrentContext
cropViewController.delegate = self
cropViewController.view.isOpaque = false
cropViewController.view.backgroundColor = .clear
cropViewController.view.isHidden = true
viewController.present(cropViewController, animated: true)

DispatchQueue.main.asyncAfter(deadline: DispatchTime.now() + 0.5) {
     cropViewController.crop()
}
````

**Comment by guoyingtao** [comment]
I will take a look at it this weekend but I am not sure if I can make it.

**Comment by guoyingtao** [comment]
I found there is already an API for cropping in Mantis but I forgot to provide a way to export CropInfo 😂
```Swift
public func getCroppedImage(byCropInfo info: CropInfo, andImage image: UIImage) -> UIImage?
```

I think if I update CropViewControllerDelegate by adding CropInfo for exporting crop information, then you can use Mantis.getCroppedImage(byCropInfo:andImage:) to crop images without showing UI
```Swift
func cropViewControllerDidCrop(_ cropViewController: CropViewController,
                                   cropped: UIImage,
                                   transformation: Transformation,
                                   cropInfo: CropInfo)
```

**Comment by Mehdi-Sohrabi0** [comment]
Wonderful, so please expose that method.

**Comment by guoyingtao** [comment]
@Mehdi-Sohrabi0 I released Mantis 1.9.0 which exposes crop info in cropViewControllerDidCrop. Then you can use `Mantis.crop(image:by:)` to get the cropped image by exposed crop info.
Please let me know if you have any questions.

**Comment by Mehdi-Sohrabi0** [comment]
tnx a lot.
